examples/hello_world/example.toml

```diff
@@ -1,8 +1,14 @@
 name = "hello_world"
 
+[application]
+manifest = "examples/hello_world/config/config.toml"
+out = "examples/hello_world/bin/config.bin"
+
 [modules]
 module = { Cargo = { cargo_manifest = "examples/hello_world/module/rust/Cargo.toml" } }
 translator = { Cargo = { cargo_manifest = "examples/translator/module/rust/Cargo.toml" } }
+# This module is only built but not actually run.
```

Comment:
How comes?

Reply:
because we can only run one application per example, currently. I think if we want to support alternative applications, we may as well turn them into separate examples, as @daviddrysdale suggested.